### ISSUE TITLE
Update $primary and web-content/$link-colour for accessibility

### ISSUE THREAD

**Issue by quinnkeast** [sourcegraph/sourcegraph#19015]
## Problem

The current `$primary` and `web-content/$link-colour/light` values slightly fail accessibility guidelines for contrast ratios.

## Proposal

Update `$primary` and `web-content/$link-colour/light` to `#1475CF`, which passes contrast guidelines.

Context can be [found in Figma](https://www.figma.com/file/IFZmcl5QoFT1KQHo6SMm4f?node-id=246:11#66145641).

**Comment by felixfbecker** [comment]
The new color would be out of our color palette. The current `$primary` is OpenColor blue 7, should we make it [OC blue 8](https://yeun.github.io/open-color/#blue-8:~:text=BLUE%208)?

**Comment by pdubroy** [comment]
@quinnkeast Is this still relevant after the redesign? If not, let's close this.

**Comment by quinnkeast** [comment]
Addressed by the redesign.
